examples/async_transcribe_local_media_file.js

```diff
@@ -28,7 +28,8 @@ const token = require('./config/config.json').access_token;
         filter_profanity: false, //Optional value available with some languages
         remove_disfluencies: false, //Optional value available with some languages
```

Comment:
Can we add spaces after the comment prefix `//` here as well as other lines that do not have a space in this file (I see lines 18-20 also have this issue?
```suggestion
        filter_profanity: false, // Optional value available with some languages
        remove_disfluencies: false, // Optional value available with some languages
```